Gemfile.lock

```diff
@@ -184,25 +184,25 @@ GEM
     turbolinks (5.2.1)
       turbolinks-source (~> 5.2)
     turbolinks-source (5.2.0)
-    tzinfo (1.2.6)
+    tzinfo (1.2.7)
```

Comment:
### [tzinfo](https://tzinfo.github.io)

**Patch** version upgrade :chart_with_upwards_trend::small_blue_diamond: 1.2.6 → 1.2.7

[_~~change-log~~, [source-code](https://github.com/tzinfo/tzinfo)_]





<details>
<summary>Commits</summary>

A change of **19** commits. See the full changes on [the compare page](https://github.com/tzinfo/tzinfo/compare/v1.2.6...v1.2.7).

These are the first 10 commits:
- (7a603da) [Update to Ruby 2.7.0.](https://github.com/tzinfo/tzinfo/commit/7a603da1baa30696e7603c5c95cc3b28dc4c1d07)
- (5f9b1cd) [`$VERBOSE = false` won't be worked since `rb_warning` is cha…](https://github.com/tzinfo/tzinfo/commit/5f9b1cd56f80b8975dbe50678d11b14811a38167)
- (83bdd0e) [Fix erroneous 'wrong number of arguments' errors on JRuby 9.…](https://github.com/tzinfo/tzinfo/commit/83bdd0ed2a701ba85d1cb36cb11833385fb02b44)
- (e0e66f8) [Fix test failures on Ruby 1.8.7.](https://github.com/tzinfo/tzinfo/commit/e0e66f8468f20c61882752b8f8c694825683bc48)
- (1d57e47) [Don't rely on lexicographic version comparisons.](https://github.com/tzinfo/tzinfo/commit/1d57e4787297c94849189dc76da3f85957bfcc44)
- (ae54281) [Fix comments relating to taint/untaint removal.](https://github.com/tzinfo/tzinfo/commit/ae54281247c8e07f913090c5942c5be019d781c9)
- (62d902f) [Test for just the non-existence of #untaint.](https://github.com/tzinfo/tzinfo/commit/62d902fc6668d9445ad4b4ec251d9281a4429adb)
- (d8556f0) [Merge pull request #112.](https://github.com/tzinfo/tzinfo/commit/d8556f0ec2f4d152d6c874ecb5da5d6d60b5747a)
- (075d5e3) [Update to JRuby 9.2.11.0.](https://github.com/tzinfo/tzinfo/commit/075d5e31b6c5ab7e53679bfe1cb899790378d8dd)
- (1f81e20) [Use https for links where available.](https://github.com/tzinfo/tzinfo/commit/1f81e20ad8b2ea76a1012e666d98d5b8e9648239)


</details>